conveyor/src/main/java/com/enfint/conveyor/exception/Refusal.java

```diff
@@ -0,0 +1,26 @@
+package com.enfint.conveyor.exception;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+
+import java.time.ZonedDateTime;
+
+@RequiredArgsConstructor
```

Comment:
This is the data-class. Use `@Data` annotation instead of @RequiredArgsConstructor